packages/client/src/client/styleguide/styled/Block.tsx

```diff
@@ -14,8 +14,20 @@ export const Block = styled.div<BlockProps>`
   ${({ theme, bg, fg }) =>
     css({
       transition: bg ? "background-color ease-out 0.15s" : undefined,
-      backgroundColor: bg && getThemeColor(theme, bg, theme.primary.base),
-      color: fg && getThemeColor(theme, fg, theme.secondary.base),
+      backgroundColor:
```

Comment:
existing stuff but I think the first attribute is the correct format (ternary) ?